core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -839,7 +839,7 @@ export class CreateServiceComponent implements OnInit {
     this.submitted = true;
     this.getData();
     this.createService();
-    this.typeOfService = 'api';
+    this.typeOfService = 'function';
```

Comment:
What's the purpose of this change?

Reply:
So by default the service type is API, whenever you open the create-service window. Since we are disabling api now, so it changed to function. The way it is changed is wrong. It has to be this way ONLY if azure is the provider.

Reply:
Done